src/pages/patientView/mutation/oncoprint/MutationOncoprint.tsx

```diff
@@ -691,7 +691,7 @@ export default class MutationOncoprint extends React.Component<
                         <Oncoprint
                             key="MutationOncoprint"
                             ref={this.oncoprintComponentRef}
-                            oncoprintRef={this.oncoprintRef}
+                            oncoprintJsRef={this.oncoprintRef}
```

Comment:
Not a big deal but now this is out of sync with this.oncoprintRef?    maybe update both or leave it?

Reply:
Renamed in [8eb47cd](https://github.com/cBioPortal/cbioportal-frontend/pull/4152/commits/8eb47cd80e85f46de223d96f55527cdfaf677d69)